link-check-config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can we move this file into `ci/` as well?

Reply:
Thanks for the review @lumjjb. Moved in affb4df65436c5a6acd3a02cd0bac4ae8e1dd865.